paths/projects/update.yaml

```
@@ -60,28 +60,96 @@ requestBody:
         type: object
         title: project/update/parameters
         properties:
+          account_id:
+            description: Required if the requesting user is a member of multiple accounts. Account ID to specify the actual account the project should be created in.
+            type: string
+            example: abcd1234
           name:
-            description: Name of the project
+            description: (Optional) Name of the project
             type: string
             example: My Android Project
           main_format:
-            description: Main file format specified by its API Extension name. Used for locale downloads if no format is specified. For API Extension names of available file formats see <a href="https://help.phrase.com/help/supported-platforms-and-formats">Format Guide</a> or our <a href="#formats">Formats API Endpoint</a>.
+            description: (Optional) Main file format specified by its API Extension name. Used for locale downloads if no format is specified. For API Extension names of available file formats see <a href="https://help.phrase.com/help/supported-platforms-and-formats">Format Guide</a> or our <a href="#formats">Formats API Endpoint</a>.
             type: string
             example: yml
           shares_translation_memory:
-            description: Indicates whether the project should share the account's translation memory
+            description: (Optional) Indicates whether the project should share the account's translation memory
             type: boolean
             example: true
           project_image:
-            description: Image to identify the project
+            description: (Optional) Image to identify the project
             type: string
             format: binary
             example: "/path/to/my/project-screenshot.png"
           remove_project_image:
-            description: Indicates whether the project image should be deleted.
+            description: (Optional) Indicates whether the project image should be deleted.
             type: boolean
-            example: 
-          account_id:
-            description: Account ID to specify the actual account the project should be created in. Required if the requesting user is a member of multiple accounts.
+            example: false
+          workflow:
+            description: (Optional) Review Workflow. "simple" / "review". <a href="https://help.phrase.com/help/advanced-review-workflow">Read more</a>
             type: string
-            example: abcd1234
+            example: review
+          machine_translation_enabled:
+            description: (Optional) Required for Autopilot and Smart Suggest
```

Just as an idea maybe a little bit of more information

```suggestion
            description: (Optional) Enable machine translation support in the project. Required for Autopilot and Smart Suggest
```